packages/components/header/template.njk

```diff
@@ -1,167 +1,87 @@
-                <button class="nhsuk-search__submit" type="submit">
-                  <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
-                    <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
-                  </svg>
-                  <span class="nhsuk-u-visually-hidden">Search</span>
-                </button>
-              </form>
-            </div>
-          </div>
-      </div>
-
-    </div>{# close nhsuk-header__container #}
-
-    {% endif -%}
-
-    {% if showNav == "true" and params.primaryLinks and showSearch == "false" %}
-
-    </div>{# close nhsuk-header__container #}
-    
-    <div class="nhsuk-navigation-container">
-      <nav class="nhsuk-navigation" id="header-navigation" role="navigation" aria-label="Primary navigation">
-        <ul class="nhsuk-header__navigation-list {%- if params.primaryLinks.length < 4 %} nhsuk-header__navigation-list--left-aligned{% endif %}">
-          {%- for item in params.primaryLinks %}
-          <li class="nhsuk-header__navigation-item {%- if item.classes %} {{ item.classes }}{% endif %}" {%- for attribute, value in item.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
-            <a class="nhsuk-header__navigation-link"  href="{{item.url}}">
-              {{item.label}}
-            </a>
-          </li>
-          {%- endfor %}
-          <li class="nhsuk-header__navigation-item nhsuk-header__navigation-item--home">
-            <a class="nhsuk-header__navigation-link"  href="/">
-              Home
-            </a>
-          </li>
-          <li class="nhsuk-mobile-menu-container">
-            <button class="nhsuk-header__menu-toggle nhsuk-header__navigation-link" id="toggle-menu" aria-expanded="false">
-              <span class="nhsuk-u-visually-hidden">Browse</span>
-              More
-              <svg class="nhsuk-icon nhsuk-icon__chevron-down" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
-                <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+{%- endif %}
+
+{%- if params.search %}
+    <div class="nhsuk-header__content" id="content-header">
+      <div class="nhsuk-header__search {%- if not params.primaryLinks %} nhsuk-header__search-no-nav{% endif %}">
+        <div class="nhsuk-header__search-wrap" id="wrap-search">
+          <form class="nhsuk-header__search-form" id="search" action="{{ params.search.action | default('https://www.nhs.uk/search/') }}" method="get" role="search">
+            <label class="nhsuk-u-visually-hidden" for="search-field">{{ params.search.label | default('Search the NHS website') }}</label>
```

Comment:
This isn’t done consistently elsewhere, but given this is a new param option, should we follow the convention of having `params.search.label.text` and `params.search.label.html`?

Reply:
Even though the text is hidden? What HTML might you place in this area?

Reply:
Dunno, but I just keep forgetting which params need an object with a `text` and which ones just need strings, so maybe it's good to try and improve consistency? I can’t think of any good use cases for html though.

Could also support `label.classes` maybe and only default to visually-hidden? But maybe that's extending scope too much?

Reply:
+1 for consistency, -1 for usefulness. Maybe an alternative option name is better, something like `visuallyHiddenLabel`. Are there other examples in the NHS Design System that do something similar?

Reply:
visuallyHiddenLabel gets my vote. 👍

Reply:
Cool. Updated to use that name.